desktop/core/src/desktop/urls.py

```diff
@@ -222,7 +222,7 @@
 ]
 
 dynamic_patterns += [
-  url(r'^admin/?', include(admin.site.urls)),
+  url(r'^admin/?', admin.site.urls),
```

Comment:
Are you sure all the admin pages work?

Reply:
following the given in https://docs.djangoproject.com/en/2.0/releases/1.9/#passing-a-3-tuple-or-an-app-name-to-include